src/Main.purs

```diff
@@ -31,6 +33,19 @@ main = Aff.launchAff_ do
     _ -> do
       log $ "Unknown arguments: " <> List.intercalate " " args
 
+generate :: List String -> Aff Unit
+generate extraArgs = do
+  case (parse extraArgs) of
+    Nothing -> do
+      log $ "Expected an integer, got: " <> List.intercalate " " extraArgs
```

Comment:
questionable error message but on par with purescript's top level output

Reply:
Rather than leaving me to guess, what error message do you want?

Reply:
Please see new commit for an attempt to improve the error message.